hack/k8s-client-go.patch

```diff
@@ -1,5 +1,5 @@
---- vendor/k8s.io/client-go/tools/leaderelection/leaderelection.go	2020-07-02 11:15:19.000000000 -0400
-+++ vendor/k8s.io/client-go/tools/leaderelection/leaderelection_new.go	2020-07-06 15:51:35.000000000 -0400
+--- ../vendor/k8s.io/client-go/tools/leaderelection/leaderelection.go	2020-07-02 11:15:19.000000000 -0400
++++ ../vendor/k8s.io/client-go/tools/leaderelection/leaderelection.go	2020-07-06 15:51:35.000000000 -0400
```

Comment:
I think this assumes it is running within `./hack`?

Here's a PR that undoes one we had in serving (if you look at the "before", it should show what we've done): https://github.com/knative/serving/pull/8531